v4.14/graph.yaml

```diff
@@ -4,7 +4,7 @@ entries:
     name: rhtas-operator
     schema: olm.package
   - entries:
-      - name: rhtas-operator.v1.2.0
+      - name: rhtas-operator.v1.2.1
```

Comment:
This Change will remove the rhtas-operator.v1.2.0 from the stable-v1.2 and replace it with rhtas-operator.v1.2.1